src/cloud/lib/hooks/useCloudApi.ts

```diff
@@ -181,8 +181,12 @@ export function useCloudApi() {
               childFoldersIds: [],
             },
           ])
+
           if (res.folder.parentFolderId != null) {
-            const parentFolder = foldersMap.get(res.folder.parentFolderId)
+            const parentFolder =
+              res.parentFolder == null
+                ? foldersMap.get(res.folder.parentFolderId)
```

Comment:
@Rokt33r We could also use `res.folder.parentFolder` if the backend does not provide `parentFolder`, and if that is null as well, we can use one from `foldersMap`, what do you think?